fcrepo-postgres.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.6"
+
+networks:
+  appnetwork:
+    driver: bridge
+
+services:
+  fcrepo:
+    container_name: fcrepo
+    build: 
+      context: ./docker/services/fcrepo
+      args:
+        FEDORA_VERSION: 4.7.5
+        FEDORA_TAG: 4.7.5
+        ModeshapeConfig: jdbc-postgresql
+        JDBCConfig: -Dfcrepo.postgresql.username=fcrepoadmin -Dfcrepo.postgresql.password=fcrepoadminpw -Dfcrepo.postgresql.host=postgres.docker.local -Dfcrepo.postgresql.port=5432 
+    image: fcrepo
+    networks:
+      appnetwork:
+        aliases:
+          - fcrepo.docker.local
+    ports:
+      - 8080:8080
+  mysql:
```

Comment:
Just an initial review pass, but I believe it's misleading to call this service `mysql` when it's actually running PostgreSQL as its database.  You should probably call the service `postgresql` instead.